security-openid-connect-quickstart/src/test/java/org/acme/security/openid/connect/BearerTokenAuthenticationTest.java

```diff
@@ -10,29 +10,42 @@
 
     KeycloakTestClient keycloakClient = new KeycloakTestClient();
 
+    static {
+        RestAssured.useRelaxedHTTPSValidation();
+    }
+
     @Test
-    public void testAdminAccess() {
+    public void testUserAccess() {
+        RestAssured.given().auth().oauth2(getAccessToken("alice"))
+                .when().get("/api/users/me")
+                .then()
+                .statusCode(200);
+
         RestAssured.given().auth().oauth2(getAccessToken("admin"))
-                .when().get("/api/admin")
+                .when().get("/api/users/me")
```

Comment:
@survivant Hmm but this drops a check that `alice` (who only has a user role) can not access `admin`-level resources